lib/Indexer/Adapter/Tolerant/Indexer/ConstantDeclarationIndexer.php

```diff
@@ -81,20 +81,26 @@ private function fromDefine(CallExpression $node, Index $index, TextDocument $do
             return;
         }
 
+        // In this case we only care about the name of the constant which is the first argument
+        $firstArgument = null;
         foreach ($node->argumentExpressionList->getChildNodes() as $expression) {
             if (!$expression instanceof ArgumentExpression) {
                 return;
             }
-            $string = $expression->expression;
-            if (!$string instanceof StringLiteral) {
-                return;
-            }
 
-            $record = $index->get(ConstantRecord::fromName($string->getStringContentsText()));
-            assert($record instanceof ConstantRecord);
-            $record->setStart(ByteOffset::fromInt($node->getStartPosition()));
-            $record->setFilePath($document->uri()->path());
-            $index->write($record);
+            $firstArgument = $expression;
+            break;
```

Comment:
i think we just needed to add the break and the original code was ok? doing it like this means dealing with NULL and adding a temporary var on line 85. if we want to make the code "cleaner" we can extract a method, but I think just adding `break` is enough :slightly_smiling_face: